lib/inferno/dsl/runnable.rb

```diff
@@ -371,6 +371,13 @@ def missing_inputs(submitted_inputs)
 
         required_inputs.map(&:to_s) - submitted_inputs.map { |input| input[:name] }
       end
+
+      def user_runnable?
+        @user_runnable ||= parent.nil? ||
+                           !parent.respond_to?(:user_runnable?) ||
```

Comment:
Is it possible for a parent not not to respond to `user_runnable?`? I don't think it should be.